recipes/r-rpymat/LICENSE.md

```diff
@@ -0,0 +1,197 @@
+Copyright 2021-2022 Zhengjia Wang. All rights reserved.
```

Comment:
This file can be deleted, the `licence_file` definition in `meta.yaml` below should pick up the file shipped in the tarball.

Reply:
I see. I have deleted this file in the latest commit

Reply:
@cbrueffer , The [CI check](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=599781&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823) suggested that `rpymat` license file was not found even though the file `LICENSE.md` is in the tarball...

Reply:
The issue is that the license is part of the repository, but not packaged as part of the tarball. The quick fix is to re-add the license here (sorry!). I see you're the upstream developer; it would be great if you could package the license file as part of your software.

Reply:
Thanks, this is actually my fault. I forgot to compile the LICENSE file in `rpymat`. Will include the LICENSE file in the next submit. However, that's not gonna happen very soon. I agree the quickest fix is to add back LICENSE.md. Once the recipe is merged into conda-forge and new version is submitted to CRAN, I will create another PR to fix the LICENSE issue in rpymat. 

Waiting for the CI to finish...

Thanks again for point this issue out : )

Reply:
No problem! Sounds like a plan.